crates/yas_core/src/tests/test_contracts/test_yas_pool.cairo

```diff
@@ -577,47 +1515,54 @@ mod YASPoolTests {
     use yas_core::contracts::yas_router::{
         YASRouter, IYASRouterDispatcher, IYASRouterDispatcherTrait
     };
-    use yas_core::tests::utils::constants::PoolConstants::{TOKEN_A, TOKEN_B, POOL_ADDRESS, WALLET};
+    use yas_core::tests::utils::constants::PoolConstants::{
+        TOKEN_A, TOKEN_B, POOL_ADDRESS, WALLET, encode_price_sqrt_1_1, encode_price_sqrt_1_10
+    };
     use yas_core::tests::utils::constants::FactoryConstants::{
         POOL_CLASS_HASH, FeeAmount, fee_amount, tick_spacing
     };
     use yas_core::contracts::yas_erc20::{
         ERC20, ERC20::ERC20Impl, IERC20Dispatcher, IERC20DispatcherTrait
     };
 
-    fn setup() -> (IYASPoolDispatcher, IERC20Dispatcher, IERC20Dispatcher) {
+
+    fn setup() -> (
+        IYASPoolDispatcher, IERC20Dispatcher, IERC20Dispatcher, IYASRouterDispatcher, i32, i32
+    ) {
         let mint_callback = deploy_mint_callback(); // 0x1
         let yas_factory = deploy_factory(OWNER(), POOL_CLASS_HASH()); // 0x2
 
         // Deploy ERC20 tokens with factory address
-        let token_0 = deploy_erc20('YAS0', '$YAS0', 4000000000000000000, OWNER()); // 0x3
-        let token_1 = deploy_erc20('YAS1', '$YAS1', 4000000000000000000, OWNER()); // 0x4
+        let token_0 = deploy_erc20(
+            'YAS0', '$YAS0', 400000000000000000000000000000000000, OWNER()
+        ); // 0x3
+        let token_1 = deploy_erc20(
+            'YAS1', '$YAS1', 400000000000000000000000000000000000, OWNER()
+        ); // 0x4
 
         set_contract_address(OWNER());
-        token_0.transfer(WALLET(), 4000000000000000000);
-        token_1.transfer(WALLET(), 4000000000000000000);
+        token_0.transfer(WALLET(), 400000000000000000000000000000000000);
+        token_1.transfer(WALLET(), 400000000000000000000000000000000000);
 
         // Give permissions to expend WALLET() tokens
         set_contract_address(WALLET());
         token_1.approve(mint_callback.contract_address, BoundedInt::max());
         token_0.approve(mint_callback.contract_address, BoundedInt::max());
 
-        let encode_price_sqrt_1_1 = FP64x96Impl::new(79228162514264337593543950336, false);
-
         let yas_pool_address = yas_factory // 0x5
             .create_pool(
-                token_0.contract_address, token_1.contract_address, fee_amount(FeeAmount::LOW)
+                token_0.contract_address, token_1.contract_address, fee_amount(FeeAmount::MEDIUM)
             );
         let yas_pool = IYASPoolDispatcher { contract_address: yas_pool_address };
 
         set_contract_address(OWNER());
-        yas_pool.initialize(encode_price_sqrt_1_1);
+        yas_pool.initialize(encode_price_sqrt_1_10());
 
         let (min_tick, max_tick) = get_min_tick_and_max_tick();
         set_contract_address(WALLET());
-        mint_callback.mint(yas_pool_address, WALLET(), min_tick, max_tick, 2000000000000000000);
+        mint_callback.mint(yas_pool_address, WALLET(), min_tick, max_tick, 3161);
```

Comment:
Does this sound like it's for a specific test case, or is it something that applies to all the tests? I'm asking because I'm not sure.

Reply:
It applies to all tests, so that the values correlate with the uni tests